Patient Registry System/PatientRegistrySystem.API/Controllers/UsersController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using PatientRegistrySystem.DB.Entities;
+using PatientRegistrySystem.Domain.Dto;
+using PatientRegistrySystem.Services;
+
+namespace PatientRegistrySystem.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class UsersController : Controller
+    {
+        public UsersController(IUserService userService)
+        {
+            _userService = userService ?? throw new ArgumentNullException(nameof(userService));
```

Comment:
you don't have to check if it's null or not, if it's null, you will get a different exception related to missing dependency.